assets/js/googlesitekit/datastore/site/info.test.js

```diff
@@ -140,6 +140,84 @@ describe( 'core/site site info', () => {
 			} );
 		} );
 
+		describe( 'getGoogleSupportURL', () => {
+			it( 'returns null if no arguments are supplied', async () => {
+				await registry.dispatch( STORE_NAME ).receiveSiteInfo( { ...baseInfo, ...entityInfo } );
+
+				const supportURL = registry.select( STORE_NAME ).getGoogleSupportURL();
+				expect( supportURL ).toEqual( null );
+			} );
```

Comment:
@asvinb looks like these tests are identical and we can convert them into table tests. So, instead of having multiple tests, we need to convert it into one test with multiple incoming args:

```javascript
it.each( [
    [
        'returns null if no arguments are supplied',
        undefined,
        null,
    ],
    [
       'returns the path with the user locale',
        { path: 'https://support.google.com/analytics/answer/1032415' },
        'https://support.google.com/analytics/answer/1032415?hl=en-US',
    ],
    ...
] )( '%s', async ( _, in, out ) => {
    await registry.dispatch( STORE_NAME ).receiveSiteInfo( { ...baseInfo, ...entityInfo } );
    const supportURL = registry.select( STORE_NAME ).getGoogleSupportURL( in );
    expect( supportURL ).toEqual( out );
} );
```

Reply:
Thanks for the tip @eugene-manuilov . Updated!